server.js

```diff
@@ -52,25 +53,23 @@ app.get("/", function (req, res) {
                 return res.status(400).json({error: error});
             else if (response && response.statusCode >= 400 && response.statusCode < 500)
                 return res.status(response.statusCode).json({error: response.statusMessage});
-            var $   = require('whacko').load(body)
-            ,   $dl = $("body div.head dl")
+            var document = (new JSDOM(body)).window.document
+            ,   dl = document.querySelector('body div.head dl')
             ,   thisURI
             ,   previousURI;
-            if ($dl) {
-                $dl.find("dt").each(function() {
-                    var $dt = $(this)
-                    txt = $dt.text()
-                    .toLowerCase()
-                    .replace(":", "")
-                    .replace("published ", "")
-                    .trim();
-                    $dd = $dt.next();
-                    if (txt === "this version") {
-                        thisURI = $dd.find('a').attr('href');
-                    }
+            if (dl) {
+                dl.querySelectorAll('dt').forEach(dt => {
+                    var txt = (dt.innerText || dt.textContent)
```

Comment:
Maybe dt.textContent first? .innerText should probably not be needed.

Reply:
resolved in https://github.com/w3c/spec-generator/pull/306/commits/b881e684b6d592f409f1c858fd688b9e59e7a30c